### PR TITLE
Added 1Forge service

### PR DESCRIPTION
**Pull request by albertodiaz**

### CODE REVIEW

**Comment by florianv** [comment]
Hey @albertodiaz, thanks for your pull request.

Can you please run `php-cs-fixer fix` to fix your code style ?

File: src/Service/Forge.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+use Exchanger\Contract\ExchangeRateQuery;
+use Exchanger\ExchangeRate;
+use Exchanger\HistoricalExchangeRateQuery;
+use Exchanger\StringUtil;
+use Exchanger\Exception\UnsupportedCurrencyPairException;
+
+/**
+ * Forge Service
+ */
+class Forge extends Service {
```
Comment (florianv) [inline src/Service/Forge.php]: Should be `1Forge` or `OneForge` ?

**Comment by albertodiaz** [comment]
Hi @florianv, I've just executed "php-cs-fixer fix".  It seems that the StyleCi analysis has failed on files that I didn't modify.

Thank you!

**Comment by florianv** [comment]
Closing this one since https://github.com/florianv/exchanger/pull/35 has been merged

**Pull request closed without merging**